api/v1alpha3/openstackmachine_types.go

```diff
@@ -126,6 +127,21 @@ type OpenStackMachineStatus struct {
 	// controller's output.
 	// +optional
 	FailureMessage *string `json:"errorMessage,omitempty"`
+
```

Comment:
Please don't update the v1alpha3. We're not going to release (at least to my understanding) any new v1alpha3 version.